<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">UrgBenri By Hokuyo Co.,LTD.</string>
  </property>
  <property name="windowIcon">
   <iconset resource="UrgBenri.qrc">
    <normaloff>:/icons/app_icon</normaloff>:/icons/app_icon</iconset>
  </property>
  <property name="locale">
   <locale language="Japanese" country="Japan"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="MutiViewerWidget" name="viewerWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>7</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_UrgBenri"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionPlugin_Information"/>
    <addaction name="separator"/>
    <addaction name="actionUBH_file_format"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionFrancais"/>
    <addaction name="actionJapanese"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionLog_Console"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="connectorsDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Connectors</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="settingsFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ConnectorPluginManager" name="deviceBox">
          <property name="styleSheet">
           <string notr="true">QTabBar::tab { height: 30px; }</string>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="usesScrollButtons">
           <bool>true</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="noConnectorLabel">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <italic>false</italic>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="../libs/ConnectorPluginManager/ConnectorPluginManager.qrc">:/ConnectorPluginManager/plug_icon</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlayControlsWidget" name="playControls" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="logConsoleDock">
   <property name="windowTitle">
    <string>Log Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="LogConsoleWidget" name="logConsole" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="filtersDock">
   <property name="windowTitle">
    <string>Filters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="FilterPluginManager" name="filtersManager" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/exit_icon</normaloff>:/icons/exit_icon</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit UrgBenri</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout_UrgBenri">
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/about_icon</normaloff>:/icons/about_icon</iconset>
   </property>
   <property name="text">
    <string>&amp;About UrgBenri</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/english_icon</normaloff>:/icons/english_icon</iconset>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="actionFrancais">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/french_icon</normaloff>:/icons/french_icon</iconset>
   </property>
   <property name="text">
    <string>&amp;Francais</string>
   </property>
  </action>
  <action name="actionJapanese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/japanese_icon</normaloff>:/icons/japanese_icon</iconset>
   </property>
   <property name="text">
    <string>日本語</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/down_icon</normaloff>:/icons/down_icon</iconset>
   </property>
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/load_icon</normaloff>:/icons/load_icon</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionUBH_file_format">
   <property name="text">
    <string>UBH file format</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/about_icon</normaloff>:/icons/about_icon</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPlugin_Information">
   <property name="icon">
    <iconset resource="UrgBenri.qrc">
     <normaloff>:/icons/plugin</normaloff>:/icons/plugin</iconset>
   </property>
   <property name="text">
    <string>Plugin Information</string>
   </property>
  </action>
  <action name="actionLog_Console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log Console</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MutiViewerWidget</class>
   <extends>QWidget</extends>
   <header>MutiViewerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectorPluginManager</class>
   <extends>QTabWidget</extends>
   <header>ConnectorPluginManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayControlsWidget</class>
   <extends>QWidget</extends>
   <header>PlayControlsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogConsoleWidget</class>
   <extends>QWidget</extends>
   <header>LogConsoleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilterPluginManager</class>
   <extends>QWidget</extends>
   <header>FilterPluginManager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../libs/ConnectorPluginManager/ConnectorPluginManager.qrc"/>
  <include location="UrgBenri.qrc"/>
 </resources>
 <connections/>
</ui>
